packages/mrgn-ui/src/components/action-box-v2/actions/lend-box/lend-box.tsx

```diff
@@ -349,6 +352,14 @@ export const LendBox = ({
     setIsLoading,
   ]);
 
+  React.useEffect(() => {
+    if (allowRefresh) {
+      refreshSelectedBanks(banks);
+      setAllowRefresh(false);
```

Comment:
@borcherd This definitely fixes the immediate issue, however I couldn't quite figure out why you went for updating a state var and then a useEffect to trigger the refresh, as opposed to just calling refreshSelectedBanks(banks) in the setIsComplete callback?

I see the `// eslint-disable-next-line react-hooks/exhaustive-deps` on the useEffect too which made me think maybe you hiut a race condition or something?